src/color.rs

```diff
@@ -2,36 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use std::f32::consts::PI;
+use std::f64::consts::PI;
```

Comment:
I don't think any color representation requires 64-bits of precision.

Reply:
One of the loose ends with this work indeed revolves around the differences in precision between f32 and f64.

On the one hand, 1/255 = 0.0(0392156862745098)..., so it'd be nice to be able to capture the full period of 16 digits.

But more importantly, some of the calculations that are done on these numbers require f64 precision in order to get a clean f32 representation to 6 decimal places (which I would think would be the minimum precision we would aim for).

PI specifically is used to convert degrees to radians.

Reply:
The point about 1/255 is still something to consider, but I otherwise found ways to avoid f64 (including the use of `f32::to_degrees`).